backend/src/main/scala/com/softwaremill/adopttapir/template/scala/EndpointsView.scala

```diff
@@ -115,17 +115,14 @@ object EndpointsView {
          |  case class Author(name: String)
          |  case class Book(title: String, year: Int, author: Author)
          |
-         |  val books = new AtomicReference(
-         |    List(
-         |      Book("The Sorrows of Young Werther", 1774, Author("Johann Wolfgang von Goethe")),
-         |      Book("Nad Niemnem", 1888, Author("Eliza Orzeszkowa")),
-         |      Book("The Art of Computer Programming", 1968, Author("Donald Knuth")),
-         |      Book("Pharaoh", 1897, Author("Boleslaw Prus"))
-         |    )
-         |)
+         |  val books = List(
+         |    Book("The Sorrows of Young Werther", 1774, Author("Johann Wolfgang von Goethe")),
+         |    Book("On the Niemen", 1888, Author("Eliza Orzeszkowa")),
+         |    Book("The Art of Computer Programming", 1968, Author("Donald Knuth")),
+         |    Book("Pharaoh", 1897, Author("Boleslaw Prus"))
+         |  )
          |${if (scalaVersion == Scala2) "}" else ""}""".stripMargin,
       Set(
-        Import("java.util.concurrent.atomic.AtomicReference"),
         Import("Library._")
```

Comment:
do we still need that import?

Reply:
Yes, since `Library.Book` and `Library.Author` is referenced in generated code, without it, the code won't compile.